ERCS/erc-7574.md

```diff
@@ -0,0 +1,191 @@
+
+  TODO: Remove this comment before submitting
+-->
+
+## Abstract
+
+<!--
+  The Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
+
+  TODO: Remove this comment before submitting
+-->
+
+
+This document outlines the step-by-step process for verifying a user's Decentralized Identity(DID) credential and issuing a Soul Bound Token (SBT) upon successful verification.
+
+We have considered granting user identity based on DID Credentials. Instead of providing the DID Credential itself within the wallet, we will use Soul Bound token(SBT) to indicate that the person has been authenticated through the Credential.
+
+When validating the Credential within the smart contract, there is a risk of exposing personal information. To address this concern, we propose a process that protects user privacy using Zero-Knowledge Proofs (Zokrates) while authenticating the user.
+
+
+## Motivation
+
+<!--
+  This section is optional.
+
+  The motivation section should include a description of any nontrivial problems the EIP solves. It should not describe how the EIP solves those problems, unless it is not immediately obvious. It should not describe why the EIP should be made into a standard, unless it is not immediately obvious.
+
+  With a few exceptions, external links are not allowed. If you feel that a particular resource would demonstrate a compelling case for your EIP, then save it as a printer-friendly PDF, put it in the assets folder, and link to that copy.
+
+  TODO: Remove this comment before submitting
+-->
+
+The anonymous nature of wallets has led to numerous issues in DeFi, Metaverse Avatars, and similar platforms. Therefore, a new interface is needed where only individuals authenticated through Credentials can receive SBTs. Moreover, only those with issued SBTs should be permitted to access specific services.
+
+
+## Specification
+
+<!--
+  The Specification section should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (besu, erigon, ethereumjs, go-ethereum, nethermind, or others).
+
+  It is recommended to follow RFC 2119 and RFC 8170. Do not remove the key word definitions if RFC 2119 and RFC 8170 are followed.
+
+  TODO: Remove this comment before submitting
+-->
+
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+
+### MetaData Interface
+
+
+NFT metadata structured by the official [ERC-721](./eip-721.md) metadata standard or the Enjin Metadata suggestions are as follows referring to OpenSea docs.
```

Comment:
> the Enjin Metadata suggestions are as follows referring to OpenSea docs

We don't allow external links (with some exceptions defined in EIP-1.) This section should contain sufficient information for an implementer to build the software, which in this case probably means some JSON schema definitions instead of calling out to the Enjin/OpenSea docs.